modules/cache/cache.ts

```diff
@@ -106,25 +108,28 @@ export const getCacheInfo = async (name: string, network: SupportedNetworks): Pr
 export const cacheGet = async (
   name: string,
   network?: SupportedNetworks,
-  expiryMs?: number
+  expiryMs?: number,
+  extendedTTL = false,
```

Comment:
That argument was added to change the cacheKey name, for keys with TTL longer than 1 day, it wouldn't be able to find them with the previous naming, as it used the current date on the name.

But now that you mention it, I think we could add a check to see if `expiryMs >= ONE_DAY_IN_MS` to use one name or the other instead of having to pass an additional argument